crud.py

```diff
@@ -0,0 +1,66 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(db: Session) -> list[models.Author]:
+    return db.query(models.Author).all()
+
+
+def get_author_by_name(db: Session, name: str) -> models.Author or None:
+    return (
+        db.query(models.Author).filter(models.Author.name == name).first()
+    )
+
+
+def create_author(db: Session, author: schemas.AuthorCreate) -> models.Author:
+    db_author = models.Author(
+        name=author.name,
+        bio=author.bio,
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_author(db: Session, author_id: int) -> models.Author or None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE]
for example: list that contain string elements and can be None has typehint Optional[list[str]]